[2020-12-01 14:43:57,257] [views.py:215] [views:get_history_monitor] [ERROR]- HTTPConnectionPool(host='10.230.13.44', port=4242): Read timed out. (read timeout=8)
[2020-12-01 14:43:57,324] [views.py:215] [views:get_history_monitor] [ERROR]- HTTPConnectionPool(host='10.230.13.44', port=4242): Read timed out. (read timeout=8)
[2020-12-01 14:43:57,333] [views.py:215] [views:get_history_monitor] [ERROR]- HTTPConnectionPool(host='10.230.13.44', port=4242): Read timed out. (read timeout=8)
[2020-12-01 14:43:57,386] [views.py:215] [views:get_history_monitor] [ERROR]- HTTPConnectionPool(host='10.230.13.44', port=4242): Read timed out. (read timeout=8)
[2020-12-01 14:43:57,387] [views.py:215] [views:get_history_monitor] [ERROR]- HTTPConnectionPool(host='10.230.13.44', port=4242): Read timed out. (read timeout=8)
[2020-12-01 14:43:57,389] [views.py:215] [views:get_history_monitor] [ERROR]- HTTPConnectionPool(host='10.230.13.44', port=4242): Read timed out. (read timeout=8)
[2020-12-01 14:44:05,307] [views.py:215] [views:get_history_monitor] [ERROR]- HTTPConnectionPool(host='10.230.13.44', port=4242): Read timed out. (read timeout=8)
[2020-12-01 14:44:05,385] [views.py:215] [views:get_history_monitor] [ERROR]- HTTPConnectionPool(host='10.230.13.44', port=4242): Read timed out. (read timeout=8)
[2020-12-01 14:44:05,399] [views.py:215] [views:get_history_monitor] [ERROR]- HTTPConnectionPool(host='10.230.13.44', port=4242): Read timed out. (read timeout=8)
[2020-12-01 14:44:05,450] [views.py:215] [views:get_history_monitor] [ERROR]- HTTPConnectionPool(host='10.230.13.44', port=4242): Read timed out. (read timeout=8)
[2020-12-01 14:44:05,456] [views.py:215] [views:get_history_monitor] [ERROR]- HTTPConnectionPool(host='10.230.13.44', port=4242): Read timed out. (read timeout=8)
[2020-12-01 14:44:05,472] [views.py:215] [views:get_history_monitor] [ERROR]- HTTPConnectionPool(host='10.230.13.44', port=4242): Read timed out. (read timeout=8)
